github/actions_workflow_runs.go

```diff
@@ -179,3 +179,17 @@ func (s *ActionsService) GetWorkflowRunLogs(ctx context.Context, owner, repo str
 	parsedURL, err := url.Parse(resp.Header.Get("Location"))
 	return parsedURL, newResponse(resp), err
 }
+
+// DeleteWorkflowRunLogs deletes all logs for a workflow run.
+//
+// GitHub API docs: https://developer.github.com/v3/actions/workflow-runs/#delete-workflow-run-logs
+func (s *ActionsService) DeleteWorkflowRunLogs(ctx context.Context, owner, repo string, runID int64) (*Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/actions/runs/%v/logs", owner, repo, runID)
+
+	req, err := s.client.NewRequest("DELETE", u, nil)
```

Comment:
use constant from net/htt package for method name
https://golang.org/pkg/net/http/#pkg-constants

Reply:
@gmlewis do we want to switch to using the constants in `net/http` throughout go-github?

Reply:
I have mixed feelings.  I know that there are a few cases in this repo where we do use the `net/http` constants, but there are very few.

I'm thinking we should proceed with this PR as-is for consistency with the rest of majority of the repo, and then maybe we could accept a separate PR from @oleg-balunenko or anyone else who wants to go through and consistently use the constants throughout the repo.